libs/blocks/mas-autoblock/mas-autoblock.js

```diff
@@ -0,0 +1,38 @@
+import { createTag, getConfig } from '../../utils/utils.js';
+import { initService } from '../merch/merch.js';
+
+export function getFragmentId(el) {
+  const { hash } = new URL(el.href);
+  const hashValue = hash.startsWith('#') ? hash.substring(1) : hash;
```

Comment:
can it be tha hash doesn't start with '#'?

Reply:
Yes, in some browser. I've seen it before but I don't remember in which one. 
And we have this check in many places in the code so I didn't want to risk.